### ISSUE TITLE
Make example editable

### ISSUE THREAD

**Issue by 89z** [dhall-lang/dhall-lang#1291]
I think it would be nice if the example here was editable:

https://dhall-lang.org

similar to this:

https://json2yaml.com/convert-yaml-to-json

**Comment by Gabriella439** [comment]
Which example are you referring to?

**Comment by 89z** [comment]
did you look at both links? that should answer the question

**Comment by SiriusStarr** [comment]
I don't see the difference; the example is editable in both?

**Comment by strawberry-choco** [comment]
I don't think that works. Yaml is basically Json with a different syntax. That is why converting in both directions are easy. In contrast, converting Json to Dhall can work, but the output would not be the same as the initial Dhall code. How is the converter supposed to know if you iterated over a list, which parts of the strings had data injected, and so on?
